src/stores/linecolors.spec.js

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert';
+import {LobsterStore} from '../stores';
+
+test('Store-Line-Colors', function() {
+  const data = [
+    '[ShardedClusterFixture:job0:configsvr:secondary0]',
+    '[ShardedClusterFixture:job0:mongos]',
+    '[ShardedClusterFixture:job0:configsvr:primary]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:configsvr:secondary1]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:mongos]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:shard0:secondary1]'
+  ];
+
+  const response = {
+    data: data.join('\n')
+  };
+  LobsterStore.processServerResponse(response);
+  var testColorMap = LobsterStore.state.colorMap;
+  assert.equal(Object.keys(testColorMap).length, 6);
+  assert.equal((':mongos]' in testColorMap), true);
```

Comment:
recommend creating an array, [':mongos]', ':shard0:primary]', ...]. then iterate through it, and assert true on each one. This is a little noisy.